lib/oneview-sdk/resource/api200/login_detail.rb

```diff
@@ -0,0 +1,24 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+module OneviewSDK
+  module API200
+    # Login Details resource implementation
+    class LoginDetail
+      BASE_URI = '/rest/logindetails'.freeze
+
+      def self.get_login_details(client)
```

Comment:
Since it is a new method you need to comment it YARD-style. (Using annotations and so on)

You can check how the methods in resource.rb are commented and follow it. You can find more information in [YARD homepage](https://yardoc.org/).